src/js/components/Skeleton/StyledSkeleton.js

```diff
@@ -36,7 +36,6 @@ export const StyledSkeleton = styled.div`
       props.theme.box.responsiveBreakpoint,
       props.theme,
     )}
-  ${(props) =>
-    props.round && roundStyle(props.round, props.responsive, props.theme)}
+  ${(props) => props.round && roundStyle(props.round, true, props.theme)}
```

Comment:
Why did we change this prop `props.responsive` to "true"?

Reply:
because there is no `props.responsive` well I guess there is `...rest` on Box but this is default to `false`

Reply:
changing back!